algebraic_connectivity_notebook.ipynb

```diff
@@ -390,7 +446,7 @@
     "    adjacency_matrix = adjacency.to_dense()\n",
     "    return int(check_MIDS(adjacency_matrix.cpu().numpy(),\n",
     "                          pred.cpu().numpy(),\n",
-    "                          sum(torch.split(data.y, num_nodes)[0].cpu().numpy())\n",
+    "                          round(sum(torch.split(data.y, num_nodes)[0].cpu().numpy()))\n",
```

Comment:
When encoding all solutions as the relative probability that a node appears in the solution, the sum might be smaller than expected due to floating point precision. For example, for a graph whose MIDS size is 3, the sum might be 2.999999345. When we give a predicted value of 3 and a target value of 2.99999345, check_MIDS would say that 3 is not less or equal to 2.99999345 and that this is not MIDS.